src/AssetConfig.php

```diff
@@ -180,7 +180,7 @@ public function load($path, $prefix = '')
             } elseif (strpos($section, self::FILTER_PREFIX) === 0) {
                 // filter section.
                 $name = str_replace(self::FILTER_PREFIX, '', $section);
-                $this->filterConfig($name, $values);
+                $this->filterConfig($name, array_merge($this->filterConfig($name), $values));
```

Comment:
Overwriting was intentional. With merging you must redefine any defaults that should be nulled out.